src/Crocodealer/Config.hs

```diff
@@ -0,0 +1,25 @@
+module Crocodealer.Config
+  ( LabelRule
+  , Label
+  , Package
+  , Config
+  ) where
+
+data Label = Label
+  { name        :: String
+  , description :: String
+  , color       :: String -- Data.Colour.sRGB24read can convert string specifications like "#00aaff" or "00aaff" 
+  }
+
+data LabelRule
+  = Create
```

Comment:
I think that this should be of the form 

```
data LabelRule = Create Label | Override <something here>
``` 

In the case of `Override`, from the example that Dmitrii has provided in the issue description, it seems like a lambda needs to be provided in order to provide the logic of what is to be done to the existing label name.
But I'm not sure how that would fit in here.

Reply:
I might need some guidance on this one.

Reply:
@kahlil29 is right, `LabelRule` should not be just enum. It should be:

```haskell
data LabelRule
    = Create Label
    | Override (Name Label) (Name Label)
```

First element of the `Override` constructor is `from` and the second is `to`. We change only names of the labels in the `Override` and then `Create` will actually do `create or update` work.